alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/processor/model/Cycle.java

```diff
@@ -0,0 +1,19 @@
+package eu.xenit.alfresco.processor.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/**
+ * Information object containing the parameters for the tracking activity.
+ */
+@Data
+@AllArgsConstructor
+public class Cycle {
+    private int txnLimit;
+    private long firstTxn;
+    private int timeIncrementSeconds;
+    private long firstCommitTime;
+
+    private long transactionId;
```

Comment:
Is there a spec I can read about these variables?
I am having a hard time figuring out what this will be used for